src/stravalib/client.py

```diff
@@ -1085,7 +1117,7 @@ def get_activity_kudos(
         )
 
         return BatchedResultsIterator(
-            entity=model.ActivityKudos,
+            entity=model.SummaryAthlete,
```

Comment:
this seems to work well. I am noticing some none fields for returned athletes. i haven't checked to see if their profiles are empty or not but these are definitely people who have used strava for some time. 
```
activities = client.get_activities(after="2024-05-01")
print(activities)

# Currently the activity object does have laps: None
first_activity = next(iter(activities))
first_activity.kudos_count  # 8
kudos = client.get_activity_kudos(first_activity.id)
for kudo in kudos:
    print(kudo)
```
Responses do consistently contain Nones and i know most atleast have their country in their profiles.

```
bound_client=<stravalib.client.Client object at 0x1306a3a90> id=None city=None country=None created_at=None firstname='Dan' lastname='F.' premium=None profile=None profile_medium=None resource_state=2 sex=None state=None summit=None updated_at=None
```